src/app/component/eco-news/components/create-edit-news/create-edit-news.component.ts

```diff
@@ -75,13 +115,15 @@ export class CreateEditNewsComponent implements OnInit, OnDestroy {
         this.form = this.createEditNewsFormBuilder.getEditForm(this.formData.value);
         this.setActiveFilters(this.formData.value);
       }
+      this.setInitialValues();
     } else {
       if (this.newsId) {
         this.fetchNewsItemToEdit();
         this.setDataForEdit();
       } else {
         this.form = this.createEditNewsFormBuilder.getSetupForm();
         this.setDataForCreate();
+        this.setInitialValues();
```

Comment:
not a problem at all, but what about dividing this logic into different methods? or come up with something better than if -else 
https://dev.to/damxipo/avoid-use-if-on-our-js-scripts-1b95